api/app/signals/apps/signals/factories/signal_user.py

```diff
@@ -0,0 +1,28 @@
+from django.contrib.auth import get_user_model
+from factory import DjangoModelFactory, Sequence, SubFactory, post_generation
+
+from signals.apps.signals.models.signal_user import SignalUser
+
+User = get_user_model()
+
+
+class UserFactory(DjangoModelFactory):
```

Comment:
We already have a UserFactory in the `users` app, please use that factory instead of this one.

Reply:
check